Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php

```diff
@@ -18,9 +19,11 @@ class BlogController extends ActionController
      * $blog is a not yet persisted clone of the original blog containing
      * the modifications
      *
-     * @Validate(param="blog", validator="FriendsOfTYPO3\BlogExample\Domain\Validator\BlogValidator")
      * @throws NoBlogAdminAccessException
      */
+    #[Validate([
```

Comment:
IMO the method argument name is missing here. Else the validator will not know which argument it should validate.

Reply:
Do you know how to pass that?

Reply:
I have found a solution in TYPO3 test sources